iceoryx_dds/include/iceoryx_dds/dds/data_reader.hpp

```diff
@@ -38,66 +38,51 @@ enum class DataReaderError : uint8_t
 constexpr char DataReaderErrorString[][64] = {
     "NOT_CONNECTED", "INVALID_RECV_BUFFER", "INVALID_DATA", "RECV_BUFFER_TOO_SMALL"};
 
+/// @brief Abstraction for DDS Data Readers.
 class DataReader
 {
   public:
-    ///
+
     /// @brief connect Connect the DataReader to the underlying DDS network.
-    ///
     virtual void connect() noexcept = 0;
 
-    ///
     /// @brief peekNextSize Get the size of the next sample if one is available.
     /// @return The size of the next sample if one is available.
-    ///
     virtual iox::cxx::optional<uint32_t> peekNextSize() = 0;
 
-    ///
     /// @brief hasSamples Checks if new samples ready to take.
     /// @return True if new samples available.
-    ///
     virtual bool hasSamples() = 0;
 
-    ///
     /// @brief take Take the next available sample from the DDS data space.
```

Comment:
Could you please correct this line. `takeNext` takes the next ....

Reply:
Removed.